src/Tribe/Ticket_Object.php

```diff
@@ -104,6 +104,15 @@ class Tribe__Tickets__Ticket_Object {
 		 */
 		public $sku;
 
+		/**
+		 * Holds the IAC setting for the ticket.
+		 *
+		 * @since TBD
+		 *
+		 * @var string
+		 */
+		public $iac = 'none';
```

Comment:
This really should be a constant from the Abstract, so we avoid having the string being passed around.

Reply:
The main code is in ET+, I was trying to avoid calling that there. For backcompat, all tickets will get a "none" setting. Open to ideas here to improve this, but this will be used as the only place we set 'none' outside of ET+.